### ISSUE TITLE
MODS

### ISSUE THREAD

**Issue by PuneetGurjar** [Benefit-Zebra/Talk#3]
I saw people saying that there is mod which open all the characters in BS IS THERE ANY MOD EXISTS ??

**Comment by Benefit-Zebra** [comment]
Yes there is a mod for 1.14 which opens all the characters for you. These characters are not opened  from store, they are inserted indirectly.

**Comment by PuneetGurjar** [comment]
Where can I get it

**Comment by Benefit-Zebra** [comment]
You can get it on Youtube. Its available there.

**Comment by PuneetGurjar** [comment]
Any mod for icons also

**Comment by Benefit-Zebra** [comment]
Are you talking about spaz icon?

**Comment by PuneetGurjar** [comment]
All icons in BS shop

**Comment by Benefit-Zebra** [comment]
> All icons in BS shop

No, we can't get them.
